sdk/dotnet/Pulumi.Automation/Events/StdoutEngineEvent.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016-2021, Pulumi Corporation
+// NOTE: The class in this file are intended to align with the serialized
+// JSON types defined and versioned in sdk/go/common/apitype/events.go
+
+namespace Pulumi.Automation.Events
+{
+    /// <summary>
+    /// StdoutEngineEvent is emitted whenever a generic message is written, for example warnings
+    /// from the pulumi CLI itself. Less common than DiagnosticEvent.
+    /// </summary>
+    public class StdoutEngineEvent
```

Comment:
I think for event naming in dotnet we should prefer full names rather than any abbreviations. StandardOutputEngineEvent or ResourceOutputsEvent

Reply:
How about the property names in `EngineEvent`? Currently they use the abbreviations since that's how the properties are named in the JSON. We could use full names for the properties in `EngineEvent` and the abbreviations in `EngineEventModel`?

Reply:
That's how I would do it!